### DISCUSSION TITLE
[제안] Issue #31의 년/월 입력 서식 구현하는 방법에 대한 건

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#91] [Ideas]
Issue #31 에서 년/월을 입력하는 서식을 만들어야 하는데, 이거 <input> 태그의 type을 month로 설정하고 CSS로 스타일만 변경시켜주면 쉽게 기능을 구현할 수 있지 않을까 싶습니다. 기획 이후에 세부적인 개발에 대해 논의할 때 이 이슈도 같이 다루어보았으면 좋겠습니다.

**Comment by Samuel-0930** [comment]
확인했습니다

**Comment by yen815** [comment]
좋습니다.

**Comment by DragonTiger92** [comment]
조원들의 확인으로 해당 discussion의 역할 완료.
